docs/source/apps/projinfo.rst

```diff
@@ -0,0 +1,312 @@
+.. _projinfo:
+
+================================================================================
+projinfo
+================================================================================
+
+.. Index:: projinfo
+
+.. only:: html
+
+    Geodetic object and coordinate operation queries
+
+Synopsis
+********
+
+    | **projinfo**
+    |    [-o formats] [-k crs|operation] [--summary] [-q]
+    |    [[--area name_or_code] | [--bbox west_long,south_lat,east_long,north_lat]]
+    |    [--spatial-test contains|intersects]
+    |    [--crs-extent-use none|both|intersection|smallest]
+    |    [--grid-check none|discard_missing|sort] [--show-superseded]
+    |    [--pivot-crs none|{auth:code[,auth:code]*}]
+    |    [--boundcrs-to-wgs84]
+    |    [--main-db-path path] [--aux-db-path path]*
+    |    [--identify]
+    |    [--c-ify] [--single-line]
+    |    {object_definition} | (-s {srs_def} -t {srs_def})
+
+    where {object_definition} or {object_definition} is a PROJ string, a
+    WKT string, an object name or a AUTHORITY:CODE
+    (where AUTHORITY is the name of a CRS authority and CODE the code of a CRS
+    found in the proj.db database).
+
+Description
+***********
+
+:program:`projinfo` is a program that can query information on a geodetic object,
+coordinate reference system (CRS) or coordinate operation, when the -s and -t
+options are specified, and display it under different formats (PROJ string, WKT string).
+
+It can also be used to query coordinate operations available between two CRS.
+
+The program is named with some reference to the GDAL `gdalsrsinfo` that offers
+partly similar services.
+
+
+The following control parameters can appear in any order:
+
+.. program:: projinfo
+
+.. option:: -o formats
+
+    formats is a comma separated combination of:
+    `all`,`default`,`PROJ`,`WKT_ALL`,`WKT2_2015`,`WKT2_2018`,`WKT1_GDAL`,`WKT1_ESRI`.
```

Comment:
In rst single backticks results in cursive text. I may be wrong, but I reckon you are going for monospaced code highlight. If that is the case, it is two backticks:   ``` ``text`` ```.

This comment is valid throughout most of the text below where there are many similar occurrences.

Reply:
Single backticks are not cursive text, but cross-references using the default domain. This may end up being cursive or bold or whatever if there is no target to be found.